script/grblcon.py

```diff
@@ -42,6 +42,34 @@ def __str__(self):
         ])
 
 
+class ADCs:
+    """ A simple object to parse and maintain ADC values
+    """
+
+    # pylint: disable=too-few-public-methods
+
+    def __init__(self, adc_string='0,0,0,0,0,0'):
+
+        # pylint: disable=invalid-name
+        self.x, self.y, self.z, self.c, self.force, self.revdiv = [
+            int(x) for x in adc_string.split(',')
+        ]
+        self.adc_string = adc_string
+
+    def __str__(self):
+        return '\n'.join([
+            'ADCs',
+            '---------',
```

Comment:
I still don't see what this function is for. Where is an `ADCs` object converted to a string? If it's unused, why not remove it?